src/components/EmailVerify.jsx

```diff
@@ -1,18 +1,29 @@
 import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useSelector } from 'react-redux/es/hooks/useSelector';
+import { loggedIn } from '../actions/index';
+import { useDispatch } from 'react-redux';
 
 function EmailVerify() {
-  
+  const navigate = useNavigate();
   const [OTP, setOTP] = useState('');
   const [otpFlag, setOtpFlag] = useState(true);
-
-  const otp = 123;
+  const emailState = useSelector((state)=> state.email.email)
+  const dispatch = useDispatch();
+  const otp = "123";
 
   const handleSubmit = (e) => {
     e.preventDefault();
-    if(OTP==otp)
+    if(OTP===otp)
     {
         setOtpFlag(true);
         setOTP('');
+        let obj = JSON.parse(localStorage.getItem(emailState));
```

Comment:
variable name should be self explanatory, avoid using obj or other type of names.